File: backend/DataAccess/DaoGenerated/システム共有/tt_afcomdocDto.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: DTO定義
//             共通ドキュメントテーブル
// 作成日　　: 2023.01.23
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tt_afcomdocDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tt_afcomdoc";
		public long docseq { get; set; }                          //ドキュメントシーケンス
		public string filenm { get; set; }                        //ファイル名
		public string? jigyocd { get; set; }                      //事業コード
		public string? title { get; set; }                        //タイトル
		public int filetype { get; set; }                         //ファイルタイプ
		public int filesize { get; set; }                         //ファイルサイズ
		public byte[] filedata { get; set; }                      //ファイルデータ
		public string? regsisyo { get; set; }                     //登録支所
		public string reguserid { get; set; }                     //登録ユーザーID
		public DateTime regdttm { get; set; }                     //登録日時
		public string upduserid { get; set; }                     //更新ユーザーID
		public DateTime upddttm { get; set; }                     //更新日時
    }
}

File: backend/BusinessService/Db/DaoGenerated/システム共有/tt_afcomdocDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             共通ドキュメントテーブル
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace JBD.GJS.Db
    Public Class tt_afcomdocDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tt_afcomdoc"
        Public Property docseq As Long                          'ドキュメントシーケンス
        Public Property filenm As String                        'ファイル名
        Public Property jigyocd As String                      '事業コード
        Public Property title As String                        'タイトル
        Public Property filetype As Integer                         'ファイルタイプ
        Public Property filesize As Integer                         'ファイルサイズ
        Public Property filedata As Byte()                      'ファイルデータ
        Public Property regsisyo As String                     '登録支所
        Public Property reguserid As String                     '登録ユーザーID
        Public Property regdttm As Date                     '登録日時
        Public Property upduserid As String                     '更新ユーザーID
        Public Property upddttm As Date                     '更新日時
    End Class
End Namespace
